models/ht_neuron.cpp

```diff
@@ -535,15 +568,29 @@ nest::ht_neuron::Parameters_::set( const DictionaryDatum& d )
 void
 nest::ht_neuron::State_::get( DictionaryDatum& d ) const
 {
-  def< double >( d, names::V_m, y_[ VM ] );      // Membrane potential
+  def< double >( d, names::V_m, y_[ V_M ] );     // Membrane potential
   def< double >( d, names::theta, y_[ THETA ] ); // Threshold
 }
 
 void
-nest::ht_neuron::State_::set( const DictionaryDatum& d, const Parameters_& )
+nest::ht_neuron::State_::set( const DictionaryDatum& d,
+  const ht_neuron& node,
```

Comment:
Several of the intrinsic channels have slow dynamics, O(1s) for I_h, e.g. If you change parameters, especially the membrane potential using `SetStatus/SetDefaults`, it will often be useful to set the state variables for these channels to their new steady-state values. Otherwise, one would have to simulate several seconds of biological time to get to equilibrium. Especially when setting model parameters at the beginning of a simulation, the new equilibration capability should be useful. When changing parameters during the simulation, on the other hand, you probably want to see how the system responds to the change (eg my voltage stepping tests and the wake-sleep transitions in HT05), so then one will not want to equilibrate.